encore/slurm_queue.py

```diff
@@ -15,18 +15,25 @@ def __init__(self, job_id, job_directory, config = None):
             self.config = dict()
 
     def get_batch_headers(self, model_plan):
-        mem_per_cpu = model_plan.get("mem_per_cpu", 6500)
-        cores_per_job= model_plan.get("mem_per_cpu", 56)
-
-        return ["#!/bin/bash", 
-           "#SBATCH --partition={}".format(self.config.get("QUEUE_PARTITION", "encore")),
-           "#SBATCH --job-name=gasp_{}".format(self.job_id), 
-           "#SBATCH --mem-per-cpu={}".format(mem_per_cpu), 
-           "#SBATCH --workdir={}".format(self.job_directory), 
-           "#SBATCH --cpus-per-task={}".format(cores_per_job),  
-           "#SBATCH --time=14-0", 
-           "#SBATCH --nodes=1",
-           "export OPENBLAS_NUM_THREADS=1"]
+        mem_per_cpu = model_plan.get("mem_per_cpu", self.config.get("JOB_MEM_PER_CPU", 6500))
```

Comment:
Updated slightly adding `model_plan` back in. If the model_plan itself does not contain `mem_per_cpu` or `cores_per_job`, it will fall back to the constants. This doesn't solve the current problem with them not being configurable -- could potentially reference it in the job builder for each out instead of here. ¯\\_(ツ)_/¯